packages/common/src/types/catalog.ts

```diff
@@ -64,9 +64,7 @@ export type CatalogTable = Pick<
 export type CatalogItem = CatalogField | CatalogTable;
 export type ApiCatalogResults = CatalogItem[];
 
-export type CatalogFieldWithAnalytics = CatalogField & {
-    analytics: CatalogAnalytics;
-};
+export type CatalogFieldWithAnalytics = CatalogField;
```

Comment:
This type can be removed completely

Reply:
ahh 🤦 ty

Reply:
here! https://github.com/lightdash/lightdash/pull/12192/commits/04d1f3c534758ac85aae01ef06555db4f4ff320d